### ISSUE TITLE
EbayPageNotice: a11y issue

### ISSUE THREAD

**Issue by darkwebdev** [eBay/evo-web#107]
This section is invisible for a11y because it should be labelled by element with id `general-status` which doesn't exist.

https://opensource.ebay.com/ebayui-core-react/main/index.html?path=/story/notices-tips-ebay-page-notice--simple-usage

<img width="597" alt="image" src="https://github.com/user-attachments/assets/4656e27b-8f7a-40e2-8dcd-c8217290cac7">

<img width="586" alt="image" src="https://github.com/user-attachments/assets/1f43eed4-abd9-4c59-bb28-b71b0552cd77">

This one is fine thou, cause it's labelled by svg that has label `Success`
https://opensource.ebay.com/ebayui-core-react/main/index.html?path=/story/notices-tips-ebay-page-notice--simple-usage-with-id
<img width="643" alt="image" src="https://github.com/user-attachments/assets/62639343-dd16-47e9-8c92-fc5a44b8f0fd">

<img width="759" alt="image" src="https://github.com/user-attachments/assets/20974c49-43e8-436f-9d65-58d1f5ae681f">

**Comment by HenriqueLimas** [comment]
We are currently always setting `aria-labelledby` independent to check the status icon is there or not. We shouldn't set the property when the header doesn't exist
